module/Entra/Microsoft.Entra/Groups/Add-EntraGroupMember.ps1

```diff
@@ -9,12 +9,24 @@ function Add-EntraGroupMember {
         [Parameter(Mandatory = $true, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, 
             HelpMessage = "Unique ID of the group.")]
         [ValidateNotNullOrEmpty()]
+        [ValidateScript({
```

Comment:
Just cast the $GroupId (or any GUID-type parameter) to [Guid] type.

function Test-Guid {
    param (
        [Parameter(Mandatory)]
        [Guid]$GuidValue
    )

    "Valid GUID: $GuidValue"
}

When non-GUID value is passed:
Test-Guid: Cannot process argument transformation on parameter 'GuidValue'. Cannot convert value "111" to type "System.Guid". Error: "Unrecognized Guid format."